locations/spiders/homedepot.py

```diff
@@ -15,7 +17,12 @@ class HomeDepotSpider(SitemapSpider):
     ]
 
     def parse_store(self, response):
-        item = LinkedDataParser.parse(response, "LocalBusiness")
+        json_ld = json.loads(
+            response.xpath(
+                '//script[@id="thd-helmet__script--storeDetailStructuredLocalBusinessData"]/text()'
+            ).extract_first()
+        )
```

Comment:
```suggestion
        json_ld = LinkedDataParser.find_linked_data(response, "LocalBusiness")
```

Reply:
thanks! that's a good suggestion

Reply:
Since were back to `LinkedDataParser.find_linked_data` and `LinkedDataParser.parse_ld` we can just do `LinkedDataParser.parse`, and still fix the opening hours later.

https://github.com/alltheplaces/alltheplaces/commit/3231e19179dcfe6aa8c5669d944e02eb6e8705d2 fixed this, before we had to remove the bad opening hours before `LinkedDataParser.parse_ld`.